internal/auth/auth.go

```diff
@@ -344,3 +352,68 @@ func persistConfiguration() error {
 	_ = os.MkdirAll(filepath.Dir(path), os.ModePerm)
 	return os.WriteFile(path, data, 0o600)
 }
+
+func RefreshCloudSession() error {
+	if globalConfig == nil {
```

Comment:
I see we are duplicating this too many times. May be just add this in `init(...)`, after `_ = loadConfiguration(..)` and remove from everywhere else. So that we can guarantee that `globalConfig` will never be `nil`